### PR TITLE
EXPANDR-9478 deprecate integrations, scripts and playbooks

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9478

## Description
Deprecating Ev1 integrations, playbooks and scripts.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ssokolovich — COMMENTED** [review]
Please leave the old descriptions\comments in place and add the "Deprection" sentence at the top.

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+- Deprecated. No available replacement.
+
+##### Extract and Enrich Expanse Indicators
+
+- Deprecated. No available replacement.
+
+##### Handle Expanse Incident
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:97]:
```suggestion
Deprecated. Use Xpanse - Alert Handler playbook.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse VM Enrich
+
+- Deprecated. No available replacement.
+
+##### Extract and Enrich Expanse Indicators
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:93]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse Unmanaged Cloud
+
+- Deprecated. No available replacement.
+
+##### Expanse VM Enrich
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:89]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse Find Cloud IP Address Region and Service
+
+- Deprecated. No available replacement.
+
+##### Expanse Unmanaged Cloud
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:85]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
+#### Integrations
+
+##### Cortex Xpanse Legacy (Deprecated)
+
+- Deprecated. Use CortexXpanse integration instead.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:5]:
```suggestion
Deprecated. Use CortexXpanse integration instead.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse Expander Feed (Deprecated)
+
+- Deprecated. Use CortexXpanse integration instead.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:9]:
```suggestion
Deprecated. Use CortexXpanse integration instead.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Scripts
+
+##### ExpanseAggregateAttributionCI
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:15]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse Enrich Cloud Assets
+
+- Deprecated. No available replacement.
+
+##### Expanse Find Cloud IP Address Region and Service
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:81]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Expanse Attribution
+
+- Deprecated. No available replacement.
+
+##### Expanse Enrich Cloud Assets
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:77]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Xpanse Incident Handling - Generic
+
+- Deprecated. No available replacement.
+
+##### Expanse Attribution
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:73]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+##### NSA - 5 Security Vulnerabilities Under Active Nation-State Attack
+
+- Deprecated. No available replacement.
+
+##### Xpanse Incident Handling - Generic
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:69]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+##### Handle Expanse Incident - Attribution Only
+
+- Deprecated. No available replacement.
+
+##### NSA - 5 Security Vulnerabilities Under Active Nation-State Attack
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:65]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Playbooks
+
+##### Expanse Load-Create List
+
+- Deprecated. No available replacement.
+
+##### Handle Expanse Incident - Attribution Only
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:61]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+##### MatchIPinCIDRIndicators
+
+- Deprecated. No available replacement.
+
+#### Playbooks
+
+##### Expanse Load-Create List
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:57]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseRefreshIssueAssets
+
+- Deprecated. No available replacement.
+
+##### MatchIPinCIDRIndicators
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:51]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpansePrintSuggestions
+
+- Deprecated. No available replacement.
+
+##### ExpanseRefreshIssueAssets
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:47]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseGenerateIssueMapWidgetScript
+
+- Deprecated. No available replacement.
+
+##### ExpansePrintSuggestions
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:43]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseEvidenceDynamicSection
+
+- Deprecated. No available replacement.
+
+##### ExpanseGenerateIssueMapWidgetScript
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:39]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseEnrichAttribution
+
+- Deprecated. No available replacement.
+
+##### ExpanseEvidenceDynamicSection
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:35]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseAggregateAttributionUser
+
+- Deprecated. No available replacement.
+
+##### ExpanseEnrichAttribution
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:31]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseAggregateAttributionIP
+
+- Deprecated. No available replacement.
+
+##### ExpanseAggregateAttributionUser
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:27]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+
+##### ExpanseAggregateAttributionDevice
+
+- Deprecated. No available replacement.
+
+##### ExpanseAggregateAttributionIP
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:23]:
```suggestion
Deprecated. No available replacement.
```

File: Packs/ExpanseV2/ReleaseNotes/1_10_56.md [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Scripts
+
+##### ExpanseAggregateAttributionCI
+
+- Deprecated. No available replacement.
+
+##### ExpanseAggregateAttributionDevice
+
+- Deprecated. No available replacement.
```
Comment (BigEasyJ) [inline Packs/ExpanseV2/ReleaseNotes/1_10_56.md:19]:
```suggestion
Deprecated. No available replacement.
```

**Review by BigEasyJ — CHANGES_REQUESTED** [review]
@ShirleyDenkberg please review

**Comment by BigEasyJ** [comment]
@capanw it might be worth it to add references to the [Cortex Xpanse](https://xsoar.pan.dev/docs/reference/integrations/cortex-xpanse) and [Feed](https://xsoar.pan.dev/docs/reference/integrations/xpanse-feed) content docs in the release notes when specifying what to use like the [Xpanse Alert - Handler playbook](https://xsoar.pan.dev/docs/reference/playbooks/xpanse---alert-handler)

@ssokolovich just to note:

- This pack has been replaced by [CortexXpanse](https://github.com/demisto/content/tree/master/Packs/CortexXpanse) pack.
- The APIs in these integrations no longer work and have been deprecated.
- We have worked internally and externally to gather information about what customers use today from this pack to port over to the EV2 pack.

**Comment by capanw** [comment]
@BigEasyJ, Thank you for adding those comment. Updated with provided suggestion.

**Comment by capanw** [comment]
@ssokolovich Re-added description along with deprecated messages.

File: Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.yml [hunk 24]
```diff
@@ -186,8 +186,10 @@ configuration:
   section: Collect
   advanced: true
   required: false
-description: The Xpanse integration for Cortex XSOAR leverages the Expander API to create incidents from Xpanse issues. It also leverages Xpanse's unparalleled view of the Internet to enrich IPs, domains and certificates using information from assets discovered by Xpanse Expander and risky flows detected by Xpanse Behavior.
-display: Cortex Xpanse Legacy
+description: Deprecated. Use Cortex Xpanse integration instead. >
+  The Xpanse integration for Cortex XSOAR leverages the Expander API to create incidents from Xpanse issues. It also leverages Xpanse's unparalleled view of the Internet to enrich IPs, domains and certificates using information from assets discovered by Xpanse Expander and risky flows detected by Xpanse Behavior.
```
Comment (ShirleyDenkberg) [inline Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.yml:190]:
```suggestion
  The Xpanse integration for Cortex XSOAR leverages the Expander API to create incidents from Cortex Xpanse issues. It also leverages Cortex Xpanse's unparalleled view of the Internet to enrich IPs, domains and certificates using information from assets discovered by Cortex Xpanse Expander and risky flows detected by Cortex Xpanse Behavior.
```

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_Find_Cloud_IP_Address_Region_and_Service.yml [hunk 25]
```diff
@@ -3,7 +3,9 @@ version: -1
 contentitemexportablefields:
   contentitemfields: {}
 name: Expanse Find Cloud IP Address Region and Service
-description: 'Subplaybook for Expanse Enrich Cloud Assets subplaybook. This playbook
+deprecated: true
+description: 'Deprecated. No available replacement. >
+  Subplaybook for Expanse Enrich Cloud Assets subplaybook. This playbook
```
Comment (ShirleyDenkberg) [inline Packs/ExpanseV2/Playbooks/playbook-Expanse_Find_Cloud_IP_Address_Region_and_Service.yml:8]:
```suggestion
  Sub-playbook for Expanse Enrich Cloud Assets sub-playbook. This playbook
```

**Comment by ShirleyDenkberg** [comment]
@ssokolovich @BigEasyJ @israelpoli @MLainer1 Doc review completed.

**Comment by ssokolovich** [comment]
> @capanw it might be worth it to add references to the [Cortex Xpanse](https://xsoar.pan.dev/docs/reference/integrations/cortex-xpanse) and [Feed](https://xsoar.pan.dev/docs/reference/integrations/xpanse-feed) content docs in the release notes when specifying what to use like the [Xpanse Alert - Handler playbook](https://xsoar.pan.dev/docs/reference/playbooks/xpanse---alert-handler)
> 
> @ssokolovich just to note:
> 
> * This pack has been replaced by [CortexXpanse](https://github.com/demisto/content/tree/master/Packs/CortexXpanse) pack.
> * The APIs in these integrations no longer work and have been deprecated, therefore many playbooks and scripts do not work as well.
> * The new APIs have had significant changes to where some features can not port over.
> * We have worked internally and externally to gather information about what customers use today from this pack to port over to the EV2 pack.

I'm with you 100% - the only reason why I asked that is if someone would like to use that script in the future or even using that today and want to understand what the script/playbook does :) Since the files will remain in our repo, I want to make sure we have all the required info about them.

**Comment by capanw** [comment]
Hello @MLainer1 @ShirleyDenkberg @israelpoli, thank you for reviewing. Updated with suggestions provided. Thanks.

**Comment by israelpoli** [comment]
Hi @capanw looks good, I'll take it to the next step hoping to merge it to master soon

thanks 🙏

**Review by israelpoli — APPROVED** [review]

**Pull request merged**